operator/pkg/nodeipam/nodesvclb.go

```diff
@@ -25,10 +25,19 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	controllerruntime "github.com/cilium/cilium/operator/pkg/controller-runtime"
+	"github.com/cilium/cilium/pkg/annotation"
 	"github.com/cilium/cilium/pkg/logging/logfields"
 )
 
-var nodeSvcLBClass = "io.cilium/node"
+const (
+	ciliumIngressLabelKey = "cilium.io/ingress"
+	ciliumGatewayLabelKey = annotation.Prefix + ".gateway" + "/owning-gateway"
+)
+
+var (
+	nodeSvcLBClass                 = annotation.Prefix + "/node"
+	nodeSvcLBMatchLabelsAnnotation = annotation.Prefix + ".nodeipam" + "/match-labels"
```

Comment:
I'm thinking that maybe a `match-node-labels` annotations might be more obvious here?

Reply:
I like this.